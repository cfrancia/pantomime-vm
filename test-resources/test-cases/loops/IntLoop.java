/**
 * Created by cfrancia on 26/09/16.
 */
public class IntLoop {

    public static void main(String[] args) {
        int total = 0;

        for (int i = 0; i < 10; i++) {
            total += i;
        }

        println(total);
    }

    public static native void println(int val);

}
