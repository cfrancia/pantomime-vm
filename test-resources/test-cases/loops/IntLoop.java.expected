OUT: 1783293664
